### PR TITLE
Bugfix each message should be stored with its timestamp

### PR DESCRIPTION
**Pull request by Jakub89**
Resolves: #627

### CODE REVIEW

File: concent_api/core/tests/test_unit_message_handlers.py [hunk 1]
```diff
@@ -11,3 +19,47 @@ def test_that_function_returns_true_when_ids_are_diffrent(self):
...
+
+
+@override_settings(
+    CONCENT_MESSAGING_TIME = 10,  # seconds
+)
+class TestMessagesStored(ConcentIntegrationTestCase):
+    def test_that_messages_are_stored_with_correct_timestamps(self):
+        compute_task_def = self._get_deserialized_compute_task_def(
+            task_id     = '1',
+            deadline    = "2017-12-01 11:00:00"
```
Comment (kbeker) [inline concent_api/core/tests/test_unit_message_handlers.py]: Style

File: concent_api/core/tests/test_unit_message_handlers.py [hunk 2]
```diff
@@ -11,3 +19,47 @@ def test_that_function_returns_true_when_ids_are_diffrent(self):
...
+        )
+
+        task_to_compute = self._get_deserialized_task_to_compute(
+            timestamp           = "2017-12-01 10:00:00",
+            compute_task_def    = compute_task_def,
```
Comment (kbeker) [inline concent_api/core/tests/test_unit_message_handlers.py]: Style

File: concent_api/core/tests/test_unit_message_handlers.py [hunk 3]
```diff
@@ -11,3 +19,47 @@ def test_that_function_returns_true_when_ids_are_diffrent(self):
...
+        )
+
+        report_computed_task = self._get_deserialized_report_computed_task(
+            timestamp = "2017-12-01 11:01:00",
+            task_to_compute = task_to_compute,
```
Comment (kbeker) [inline concent_api/core/tests/test_unit_message_handlers.py]: Style

File: concent_api/core/tests/test_unit_message_handlers.py [hunk 4]
```diff
@@ -11,3 +19,47 @@ def test_that_function_returns_true_when_ids_are_diffrent(self):
...
+
+        store_subtask(
+            task_id              = task_to_compute.compute_task_def['task_id'],
+            subtask_id           = task_to_compute.compute_task_def['subtask_id'],
+            provider_public_key  = provider_public_key,
+            requestor_public_key = requestor_public_key,
+            state                = Subtask.SubtaskState.FORCING_REPORT,
+            next_deadline        = int(task_to_compute.compute_task_def['deadline']) + settings.CONCENT_MESSAGING_TIME,
+            task_to_compute      = task_to_compute,
+            report_computed_task = report_computed_task,
```
Comment (kbeker) [inline concent_api/core/tests/test_unit_message_handlers.py]: Style

File: concent_api/core/tests/test_unit_message_handlers.py [hunk 5]
```diff
@@ -11,3 +19,47 @@ def test_that_function_returns_true_when_ids_are_diffrent(self):
     def test_that_function_returns_false_when_ids_are_the_same(self):
         response = are_items_unique([1, 2, 3, 2, 5])
         self.assertFalse(response)
+
+
+@override_settings(
+    CONCENT_MESSAGING_TIME = 10,  # seconds
```
Comment (kbeker) [inline concent_api/core/tests/test_unit_message_handlers.py]: Style

File: concent_api/core/tests/test_unit_message_handlers.py (hunk 1 above)
Comment (Jakub89) [reply concent_api/core/tests/test_unit_message_handlers.py]: Done

File: concent_api/core/tests/test_unit_message_handlers.py (hunk 2 above)
Comment (Jakub89) [reply concent_api/core/tests/test_unit_message_handlers.py]: Done

File: concent_api/core/tests/test_unit_message_handlers.py (hunk 3 above)
Comment (Jakub89) [reply concent_api/core/tests/test_unit_message_handlers.py]: Done

File: concent_api/core/tests/test_unit_message_handlers.py (hunk 4 above)
Comment (Jakub89) [reply concent_api/core/tests/test_unit_message_handlers.py]: Done

File: concent_api/core/tests/test_unit_message_handlers.py (hunk 5 above)
Comment (Jakub89) [reply concent_api/core/tests/test_unit_message_handlers.py]: Done

File: concent_api/core/tests/test_unit_message_handlers.py [hunk 6]
```diff
@@ -11,3 +19,47 @@ def test_that_function_returns_true_when_ids_are_diffrent(self):
...
+            task_id=task_to_compute.compute_task_def['task_id'],
+            subtask_id=task_to_compute.compute_task_def['subtask_id'],
+            provider_public_key=provider_public_key,
+            requestor_public_key=requestor_public_key,
+            state=Subtask.SubtaskState.FORCING_REPORT,
+            next_deadline=int(task_to_compute.compute_task_def['deadline']) + settings.CONCENT_MESSAGING_TIME,
+            task_to_compute=task_to_compute,
+            report_computed_task=report_computed_task,
+        )
+        stored_task_to_compute = StoredMessage.objects.get(type=2002)
```
Comment (dybi) [inline concent_api/core/tests/test_unit_message_handlers.py]: please use ~ `messsage.TYPE`

File: concent_api/core/tests/test_unit_message_handlers.py [hunk 7]
```diff
@@ -11,3 +19,47 @@ def test_that_function_returns_true_when_ids_are_diffrent(self):
...
         response = are_items_unique([1, 2, 3, 2, 5])
         self.assertFalse(response)
+
+
+@override_settings(
+    CONCENT_MESSAGING_TIME=10,  # seconds
+)
+class TestMessagesStored(ConcentIntegrationTestCase):
+    def test_that_messages_are_stored_with_correct_timestamps(self):
+        compute_task_def = self._get_deserialized_compute_task_def(
```
Comment (dybi) [inline concent_api/core/tests/test_unit_message_handlers.py]: please extract creation of messsages to some function (i.e. `setUp`)

File: concent_api/core/tests/test_unit_message_handlers.py [hunk 8]
```diff
@@ -11,3 +19,47 @@ def test_that_function_returns_true_when_ids_are_diffrent(self):
...
+            task_id='1',
+            deadline="2017-12-01 11:00:00"
+        )
+
+        task_to_compute = self._get_deserialized_task_to_compute(
+            timestamp="2017-12-01 10:00:00",
```
Comment (dybi) [inline concent_api/core/tests/test_unit_message_handlers.py]: please extract it to variable/filed ans reuse

File: concent_api/core/tests/test_unit_message_handlers.py [hunk 9]
```diff
@@ -11,3 +19,47 @@ def test_that_function_returns_true_when_ids_are_diffrent(self):
...
+            compute_task_def=compute_task_def,
+        )
+
+        report_computed_task = self._get_deserialized_report_computed_task(
+            timestamp="2017-12-01 11:01:00",
```
Comment (dybi) [inline concent_api/core/tests/test_unit_message_handlers.py]: please extract it to variable/filed ans reuse

**Pull request merged**